src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -362,115 +356,127 @@
+                                     :tx-type tx-type
                                      :receiver? true}))
-         request-params [transaction-type-param
-                         from-address
-                         to-address
-                         amount-in
-                         token-id
-                         to-token-id
-                         disabled-from-chain-ids
-                         disabled-to-chain-ids
-                         network-preferences
-                         gas-rates
-                         from-locked-amount]]
-     {:db            (cond-> db
-                       :always (assoc-in [:wallet :ui :send :amount] amount)
-                       :always (assoc-in [:wallet :ui :send :loading-suggested-routes?] true)
-                       :always (assoc-in [:wallet :ui :send :sender-network-values]
-                                sender-network-values)
-                       :always (assoc-in [:wallet :ui :send :receiver-network-values]
-                                receiver-network-values)
-                       :always (assoc-in [:wallet :ui :send :suggested-routes-call-timestamp] now)
-                       :always (update-in [:wallet :ui :send] dissoc :network-links)
-                       token   (assoc-in [:wallet :ui :send :token] token))
-      :json-rpc/call [{:method     "wallet_getSuggestedRoutes"
-                       :params     request-params
-                       :on-success (fn [suggested-routes]
-                                     (rf/dispatch [:wallet/suggested-routes-success suggested-routes
-                                                   now]))
-                       :on-error   (fn [error]
-                                     (rf/dispatch [:wallet/suggested-routes-error error])
-                                     (log/error "failed to get suggested routes"
-                                                {:event  :wallet/get-suggested-routes
-                                                 :error  error
-                                                 :params request-params}))}]})))
+         params [{:uuid                 (str (random-uuid))
+                  :sendType             transaction-type
+                  :addrFrom             from-address
+                  :addrTo               to-address
+                  :amountIn             amount-in
+                  :amountOut            amount-out
+                  :tokenID              token-id
+                  :toTokenID            to-token-id
+                  :disabledFromChainIDs disabled-from-chain-ids
+                  :disabledToChainIDs   disabled-to-chain-ids
+                  :gasFeeMode           gas-rates
+                  :fromLockedAmount     from-locked-amount
+                  :username             (:username args)
+                  :publicKey            (:publicKey args)
+                  :packID               (:packID args)}]]
+     {:db            (update-in db
```

Comment:
You can add a check for nil values here, like https://github.com/status-im/status-mobile/pull/20856/files#diff-089fb8d72eefa455e3dc6437b0bc11c3ca3cc0fb4063e20f129b09929aeae3a9R453-R454